app.rb

```diff
@@ -0,0 +1,65 @@
+require './lib/models'
+
+require 'bcrypt'
+require 'pg'
+require 'sinatra/base'
+require 'sinatra/flash'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+  register Sinatra::Flash
+
+  get '/' do
+    @user = User.get(session[:id])
+    @peeps = Peep.all(:order => [:id.desc])
+    erb(:index)
+  end
+
+  get '/users/new' do
+    erb(:'users/new')
+  end
+
+  post '/users' do
+    user = User.create(
+      peeper: params['name'],
+      username: params['username'],
+      email: params['email'],
+      password: BCrypt::Password.create(params['password'])
```

Comment:
Nice effort with the password encryption.  I'd probably refactor this to the User class as part of the create method though.